### PR TITLE
Update rubygem-foreman_puppet to 4.1.0

### PR DESCRIPTION
**Pull request by Ron-Lavi**

### CODE REVIEW

**Comment by evgeni** [comment]
please pick https://github.com/theforeman/foreman-packaging/commit/3e4c0f1924a8760df55b68a0fe826f48f34c3b32 too, then it will build fine

**Comment by Ron-Lavi** [comment]
Thanks @evgeni, updated, let's see if it builds now.

File: packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec [hunk 1]
```diff
@@ -6,10 +6,10 @@
 %global gem_name foreman_puppet
 %global plugin_name puppet
 %global foreman_min_version 3.0
-%global release 1
+%global release 2
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:9]:
```suggestion
%global release 1
```

File: packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec [hunk 2]
```diff
@@ -114,6 +115,14 @@ cp -a .%{gem_dir}/* \
 %{foreman_plugin_log}
 
 %changelog
+* Sun Dec 25 2022 Ron Lavi <1ronlavi@gmail.com> 4.1.0-1
+- Update to 4.1.0
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_puppet/rubygem-foreman_puppet.spec:119]:
```suggestion
- Update to 4.1.0

```

**Review by evgeni — APPROVED** [review]

**Pull request merged**